components/molecule/coachmark/demo/index.js

```diff
@@ -1,2 +1,33 @@
-import MoleculeCoachmark from 'components/molecule/coachmark/src'
-export default () => <MoleculeCoachmark />
+import {Article, H1, H2, Paragraph} from '@s-ui/documentation-library'
+
+import ArticleDefaultTooltipVariants from './ArticleDefaultTooltipVariants.js'
+
+import './index.scss'
+
+const Demo = () => {
+  return (
+    <div className="sui-StudioPreview">
+      <div className="sui-StudioPreview-content sui-StudioDemo-preview">
+        <H1>Coachmark</H1>
+        <Paragraph>
+          Wrapper of the react joyride library to make tours within the apps.
+        </Paragraph>
+
+        <Article className={`molecule-coachmark-section`}>
+          <H2>Props</H2>
+          <Paragraph>
+            All the props received by the coachmark component are the same
+            accepted by the react joyride library,{' '}
+            <a href="https://docs.react-joyride.com/" target="_blank">
+              please refer to the original documentation to get more details
+            </a>
+          </Paragraph>
```

Comment:
perhaps a string literal will help there? 

```suggestion
          <Paragraph>
            {`All the props received by the coachmark component are the same
            accepted by the react joyride library,
            <a href="https://docs.react-joyride.com/" target="_blank">
              please refer to the original documentation to get more details
            </a>`}
          </Paragraph>
```

Reply:
It seems that the link is not rendered as a link in that way :(

Reply:
oh you're right! didn't think about the markup inside 😅